packages/ui/src/multiplying-architecture/KaotoBridge.tsx

```diff
@@ -50,9 +51,8 @@ interface KaotoBridgeProps {
 
 export const KaotoBridge = forwardRef<EditorApi, PropsWithChildren<KaotoBridgeProps>>((props, forwardedRef) => {
   const eventNotifier = EventNotifier.getInstance();
-  const sourceCodeContext = useContext(SourceCodeContext);
```

Comment:
We can spare the `sourceCode` from the provider since we're starting with an empty string which later on gets updated by the file content.